seal5/backends/riscv_register_info/templates/registers_tablegen.mako

```diff
@@ -0,0 +1,48 @@
+// class RVInst_${name}<dag outs, dag ins> : RVInst<outs, ins, "cv.add.h", "$rd, $rs1, $rs2", [], InstFormatOther> {
+class RVInst_${name}<dag outs, dag ins> : Instruction, Sched<${sched_str}> {
+    // General
+    let Namespace = "RISCV";
+    let Size = ${xlen // 8};
```

Comment:
floor is surely safe, in that a size other than 32 or 64 seems unlikely, and even DSP-ish 24 is still sized in bytes. But maybe an assert of the assumption would be documentary? (size % 8 == 0)